### PR TITLE
[Frontend+Backend] Make the Manage FAQ Page

### PR DESCRIPTION
**Pull request by Aman16-ai**
## Issue that this pull request solves

 Closes: #677 

### Brief description of what is fixed or changed

- Fetching all faqs from backend showing in admin faq component

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Screenshot (761)

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Review by Kajol-Kumari — COMMENTED** [review]
We need to have the edit & delete icon/btn here as well. So do you have any idea on where should we put them so the ui doesn't get clumsy?

**Comment by Aman16-ai** [comment]
> We need to have the edit & delete icon/btn here as well. So do you have any idea on where should we put them so the ui doesn't get clumsy?
Screenshot (762)

can we try this?

File: frontend/src/pages/Admin/Components/Faq/ManageFaq/manage-faq.module.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.faq {
+    width: 80%;
+    margin: 0em auto 1em auto;
+    display: flex;
+    flex-direction: column;
+    height: 33rem;
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Faq/ManageFaq/manage-faq.module.scss:6]: can we please use some more height as a lot of empty spaces are left un-necessarily

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Aman16-ai please add those button as well. It would be easier to proceed further :)

**Comment by Aman16-ai** [comment]
Screenshot (765)

**Without Hover**
Screenshot (766)

**With Hover **
Screenshot (767)

**Review by Kajol-Kumari — APPROVED** [review]
Nice and Clean!
LGTM 👍

**Pull request merged**